src/hdrgen.d

```diff
@@ -2459,7 +2454,15 @@ public:
 
     override void visit(DeclarationExp e)
     {
-        e.declaration.accept(this);
+        if (auto v = e.declaration.isVarDeclaration())
+        {
+            buf.writeByte('(');
+            visitVarDecl(v, false);
+            buf.writeByte(';');
+            buf.writeByte(')');
```

Comment:
This is a change for debugging. It's not a bugfix.

This PR also changes `visit(ExpStatement s)` function. After that change, `visit(DeclarationExp e)` will never be reachable from any normal dmd execution. So, this strange output will be shown only when we'll directly use `DeclarationExp.toChars()` for debugging.

While inliner debugging, I repeatedly prints the expanded expression tree. For that, the modified output format for `DeclarationExp` will help me.